website/docs/reference/integrations/slack.md

```diff
@@ -5,7 +5,7 @@ title: Slack (deprecated)
 
 :::caution Deprecation notice
 
-This Slack integration is deprecated and will be removed in a future release. We recommend using the new [Slack App](./slack-app.md) integration instead.
+This Slack integration is deprecated and will be removed in a future release. We recommend using the new [Slack App](./slack-app) integration instead.
```

Comment:
This is for another PR, but it feels strange that `/integration/slack` is the old link. We could probably rename this page to `legacy-slack` and the other one to `/slack`

Reply:
Yes good point. I want to have a PR that looks at deprecations in general, removing them from the navigation etc. and doing a better job at guiding users towards the new functionality. I'll add this note to that project.